docs/start/getting-started/fragments/ios/data-model.md

```diff
@@ -146,8 +146,7 @@ Call the `runMutation()`, `runQuery()`, and `subscribe()` methods from your app
 You can open the AWS console for you to run Queries, Mutation, or Subscription against you new API at any time directly by running the following command:
 
 ```bash
-$ amplify console api
-> GraphQL               ##Select GraphQL
+amplify console api
 ```
 
-This will open the AWS AppSync console for you to run Queries, Mutations, or Subscriptions at the server and see the changes in your client app.
+When prompted, select GraphQL. This will open the AWS AppSync console for you to run Queries, Mutations, or Subscriptions at the server and see the changes in your client app.
```

Comment:
user studies have shown that people don't read much outside of codeblocks. we should make these either bold or retain them in comments